terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java

```diff
@@ -335,4 +381,22 @@ public void setColumns(Map<String, Map<Locale, String>> cols) {
     public void afterPropertiesSet() {
         Assert.notNull(codeListTable, "codeListTable is not initialized!");
     }
+
+    /**
+     * Gets the default locale as fallback.<br>
+     * @return the default locale as fallback
+     */
+    public Locale getFallbackTo() {
+        return fallbackTo;
+    }
```

Comment:
- [ ] I think that no need to povide Getter. What kind of scenes do you intend to use?

Reply:
I thought it to provide to ability of access `fallbackTo` property from sub classes.

Reply:
@btkitsunedukam It is enough to provide protected field.

Reply:
Modified by 662164b